libs/state/src/lib/medical-illustration/medical-illustration.state.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { Action, State, StateContext } from '@ngxs/store';
+import { parse, ParseResult } from 'papaparse';
+
+import { SetActiveNode, SetMapping, SetUri } from './medical-illustration.actions';
+import { MedicalIllustrationModel } from './medical-illustration.model';
+
+export type MedicalIllustrationContext = StateContext<MedicalIllustrationModel>;
+
+@State<MedicalIllustrationModel>({
+  name: 'medicalIllustration',
+  defaults: {},
+})
+@Injectable()
+export class MedicalIllustrationState {
+  @Action(SetUri)
+  setUri({ setState }: MedicalIllustrationContext, { url }: SetUri) {
+    setState({ url });
+  }
+
+  @Action(SetActiveNode)
+  setActiveNode({ patchState }: MedicalIllustrationContext, { node }: SetActiveNode) {
+    patchState({ node });
+  }
+
+  @Action(SetMapping)
```

Comment:
Add `{ cancelUncompleted: true }` as the options argument to `@Action(action, options)`. That way it will always have the latest mapping